docs/references/backend/types/paginated-resource-response.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: PaginatedResourceResponse
+description: An interface that describes the response of a method that returns a paginated list of resources.
+---
+
+# `PaginatedResourceResponse`
+
+An interface that describes the response of a method that returns a paginated list of resources.
+
+## Properties
+
+| Properties | Type | Description |
+| --- | --- | --- |
+| `data` | `T[]` |  An array that contains the fetched data. |
```

Comment:
Do we provide a generic function that the user can pass their own object to define the types from their fetched data? 

otherwise it should be `unknown[]` or `any[]`

Reply:
Nevermind I see the example below